js/models/messages.js

```diff
@@ -1125,38 +1111,43 @@
 
       const quoteWithData = await loadQuoteData(this.get('quote'));
       const previewWithData = await loadPreviewData(this.get('preview'));
+      const chatMessage = new libsession.Messages.Outgoing.ChatMessage({
+        body,
+        timestamp: this.get('sent_at'),
+        attachments,
+        quote: quoteWithData,
+        preview: previewWithData,
+        expireTimer: this.get('expireTimer'),
+      });
 
       // Special-case the self-send case - we send only a sync message
       if (number === this.OUR_NUMBER) {
-        const dataMessage = await textsecure.messaging.getMessageProto(
-          number,
-          body,
-          attachments,
-          quoteWithData,
-          previewWithData,
-          this.get('sent_at'),
-          this.get('expireTimer'),
-          profileKey
-        );
-        return this.sendSyncMessageOnly(dataMessage);
+        this.trigger('pending');
+        await this.markMessageSyncOnly();
+        const syncMessage = libsession.Utils.SyncMessageUtils.from(chatMessage);
```

Comment:
we can let the queue handle sync messages so we can fallback to the case below

Reply:
fixed